--- orig/atl-init-jira	2017-07-27 10:43:15.164967500 -0400
+++ atl-init-jira	2017-07-27 10:53:16.079396844 -0400
@@ -292,6 +292,12 @@
     atl_log "Cleaning up"
     rm -rf "$(atl_tempDir)"/installer* >> "${ATL_LOG}" 2>&1
 
+    atl_log "Setting JVM heap settings"
+    sed -i \
+        -e "s/^JVM_MINIMUM_MEMORY=.*/JVM_MINIMUM_MEMORY=\"${ATL_JVM_MIN_MEMORY}\"/" \
+        -e "s/^JVM_MAXIMUM_MEMORY=.*/JVM_MAXIMUM_MEMORY=\"${ATL_JVM_MAX_MEMORY}\"/" \
+	${ATL_JIRA_INSTALL_DIR}/bin/setenv.sh
+
     chown -R "${ATL_JIRA_USER}":"${ATL_JIRA_USER}" "${ATL_JIRA_INSTALL_DIR}" 
 
     atl_log "${ATL_JIRA_SHORT_DISPLAY_NAME} installation completed"
